rubicon/objc/runtime.py

```diff
@@ -186,6 +186,13 @@ class objc_property_t(c_void_p):
     pass
 
 
+class objc_property_attribute_t(Structure):
```

Comment:
I don't understand why this was moved?

Reply:
e344fa50b878f766336e5c5780897512212268ea added the declaration for `class_addProperty`, whose signature includes an `objc_property_attribute_t`. `class_addProperty` is declared together with all other `class_*` functions, above the previous location of `objc_property_attribute_t`, so I had to move the type up so the new function declaration could use it.